### PR TITLE
Prepare DAO dashboard for mainnet

### PR DESCRIPTION
**Pull request by Siegrift**
Fixes https://github.com/api3dao/api3-dao-dashboard/issues/192

@bbenligiray don't forget to do this after you deploy to mainnet and create the ENS name https://github.com/api3dao/api3-dao-dashboard/issues/148

### CODE REVIEW

File: src/contract-deployments/mainnet-dao.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Siegrift) [inline src/contract-deployments/mainnet-dao.json:1]: @bbenligiray Feel free to commit the deployed addresses here, these are wrong atm.

File: .env.example [hunk 2]
```diff
@@ -9,7 +9,7 @@ RINKEBY_PROVIDER_URL=
 # (https://create-react-app.dev/docs/adding-custom-environment-variables/)
 REACT_APP_NODE_ENV=development
 REACT_APP_RINKEBY_PROVIDER_URL=
-REACT_APP_MAIN_PROVIDER_URL=
+REACT_APP_MAINNET_PROVIDER_URL=
```
Comment (Siegrift) [inline .env.example:12]: TODO(for me): I need to set this on Fleek.

**Review by Siegrift — COMMENTED** [review]
@bbenligiray FYI: After this is merged to `main`. We will merge `main` branch to the `production` one (such that it's a separate Fleek app for production and staging)

File: .env.example (hunk 2 above)
Comment (Siegrift) [reply .env.example:12]: Done.

File: src/contract-deployments/mainnet-dao.json (hunk 1 above)
Comment (Siegrift) [reply src/contract-deployments/mainnet-dao.json:1]: I'll commit them.

File: src/chain-data/helpers.ts [hunk 3]
```diff
@@ -32,14 +32,7 @@ export const getNetworkData = async (provider: ethers.providers.Web3Provider | n
...
-  let networkName = goResponse[GO_RESULT_INDEX].network.name;
-  // NOTE: The localhost doesn't have a name, so set any unknown networks
-  // to localhost. The network name is needed to display the "Unsupported Network"
-  // message to the user if required and in "connected to" status panel.
-  if (networkName === 'unknown') networkName = 'localhost';
-  // Convert "homestead" to mainnet for convenience
-  if (networkName === 'homestead') networkName = 'mainnet';
-
+  let networkName = updateNetworkName(goResponse[GO_RESULT_INDEX].network.name);
   const networdData: Partial<ChainData> = {
```
Comment (bbenligiray) [inline src/chain-data/helpers.ts:36]: `netword` :thinking:

File: src/chain-data/helpers.ts [hunk 4]
```diff
@@ -32,14 +32,7 @@ export const getNetworkData = async (provider: ethers.providers.Web3Provider | n
...
 
-  let networkName = goResponse[GO_RESULT_INDEX].network.name;
-  // NOTE: The localhost doesn't have a name, so set any unknown networks
-  // to localhost. The network name is needed to display the "Unsupported Network"
-  // message to the user if required and in "connected to" status panel.
-  if (networkName === 'unknown') networkName = 'localhost';
-  // Convert "homestead" to mainnet for convenience
-  if (networkName === 'homestead') networkName = 'mainnet';
-
+  let networkName = updateNetworkName(goResponse[GO_RESULT_INDEX].network.name);
```
Comment (bbenligiray) [inline src/chain-data/helpers.ts:35]:
```suggestion
   const networkName = updateNetworkName(goResponse[GO_RESULT_INDEX].network.name);
```

File: src/chain-data/helpers.ts (hunk 3 above)
Comment (bbenligiray) [inline src/chain-data/helpers.ts:36]:
```suggestion
  const networkData: Partial<ChainData> = {
```

Comment (bbenligiray) [inline src/chain-data/helpers.ts:36]: Eh couldn't create the suggestion for this but looks like a typo

**Review by andreogle — APPROVED** [review]

**Review by bbenligiray — COMMENTED** [review]
l/g/t/m (though check my addresses I guess)

**Pull request merged**